frame/bags-list/remote-tests/src/lib.rs

```diff
@@ -32,12 +32,18 @@ pub mod try_state;
 ///
 /// For example, this can be the `Runtime` type of the Polkadot runtime.
 pub trait RuntimeT<I: 'static>:
-	pallet_staking::Config + pallet_bags_list::Config<I> + frame_system::Config
+	pallet_staking::Config
+	+ pallet_stake_tracker::Config
```

Comment:
why is this needed?

Reply:
To have access to RW methods on the list, staking has read-only access from now on.

Reply:
redundant now, we will keep exposing unsafe methods.